### ISSUE TITLE
Can we concatenate strings in dentaku?

### ISSUE THREAD

**Issue by rcanand** [rubysolo/dentaku#78]
I can make the evaluated expression return a string - like `calc.evaluate("foo", foo: 'abc')` returns the string `"abc"`. Can I concat strings as well? I tried:

`calc.evaluate("foo + bar", foo:"abc", bar: "def")`

but it returns `0`, and not `"abcdef"`. Is there some other function that can do this?

I added a custom function for this - is it ok to use dentaku for string manipulations like this?

```
calc.add_function(:concat, :string,  ->(str1, str2) { str1 + str2 })
calc.evaluate("concat(foo, bar)", foo: 'abc', bar: 'def')
=> "abcdef"

```

**Comment by rubysolo** [comment]
Interestingly enough, I had a branch with this feature half done -- I have finished it and pushed it up: d881952

`CONCAT` is now available, in addition to a few other string manipulation functions.

**Comment by rcanand** [comment]
I am using the current gem version (2.0.7), but it does not recognize the concat function.

**Comment by rubysolo** [comment]
It was previously only on master, but I have just released v2.0.8 which includes `CONCAT` support, so it should be available on rubygems in the next few minutes.

**Comment by rcanand** [comment]
Thanks, @rubysolo !
